editor/src/clj/editor/gui.clj

```diff
@@ -2626,6 +2625,10 @@
 (defn register-gui-scene-loader! [loader]
   (swap! custom-gui-scene-loaders conj loader))
 
+;; Used by tests
+(defn clear-custom-gui-scene-loaders-for-tests! []
+  (reset! custom-gui-scene-loaders ()))
+
```

Comment:
This is a hacky workaround for the fact that we do not currently have any concept of "unloading" an editor plugin. 😞